### ISSUE TITLE
Over shooting x-axis for probing 

### ISSUE THREAD

**Issue by itsnotlevel1** [winder/Universal-G-Code-Sender#2287]
### Version

2.0.19

### Hardware / Firmware

FluidNC

### What happened

I have been trying to make the probe function work and it actually only worked ONCE...lol. Now whenever using the xyz probe function: Z works great....X works great.....but when the spindle moves over the touch the Y side, it over shoots the probe by at least 1 inch.....??? Video below. 

https://drive.google.com/file/d/1dL07ceMukKIUcAebgRDHZZhMMcU-098Y/view?usp=sharing

### How to reproduce

Just run probe again. 

### Operating System

Linux Mint 20.2

### Anything else

everytime

**Comment by breiler** [comment]
Can you screenshot your your probe settings.

**Comment by itsnotlevel1** [comment]
![Screenshot from 2023-08-19 13-36-59](https://github.com/winder/Universal-G-Code-Sender/assets/59421087/14364f8e-a873-4e76-a4e8-863aa188e431)
![Screenshot from 2023-08-19 13-37-19](https://github.com/winder/Universal-G-Code-Sender/assets/59421087/fe41c446-bac4-47c1-9fa9-af00059e7112)

**Comment by breiler** [comment]
Thanks, I see the problem now when trying the same settings on my machine. It was hardly noticeable when using "smaller" values so I never caught this. It is getting late here, so I'll take a look at it tomorrow morning.

**Comment by itsnotlevel1** [comment]
Yes sir .... Please take your time, it's the weekend. 😁. Btw.... Those are imperial measurements just incase I didn't specify. Thanks!

**Comment by breiler** [comment]
I think that I figured it out. It is available in the latest nightly build:
https://github.com/winder/Universal-G-Code-Sender#downloads

**Comment by itsnotlevel1** [comment]
Yayyyy!! You are a coding god! Thank you for your time.

**Comment by breiler** [comment]
Excellent, thanks for the feedback. Is it ok to close the issue?

**Comment by itsnotlevel1** [comment]
Yes, I believe so.

Paul

-------- Original Message --------
